print-radius.c

```diff
@@ -541,6 +609,68 @@ print_attr_string(netdissect_options *ndo,
            data++;
            length--;
         break;
+      case EAP_MESSAGE:
+           type = *data;
+           len = EXTRACT_16BITS(data+2);
+           ND_PRINT((ndo, "\n\t\t %s (%u), id %u, len %u",
+                  tok2str(eap_code_values, "unknown", type),
+                  type, *(data+1), len));
+           if ((type == 1) || (type == 2)) /* For EAP_REQUEST and EAP_RESPONSE only */
+           {
+              subtype = *(data+4);
+              ND_PRINT((ndo, "\n\t\t Type %s (%u)",
+              tok2str(eap_type_values, "unknown", *(data+4)),
+                      *(data + 4)));
+              switch (subtype)
+              {
+                 case EAP_TYPE_IDENTITY:
+                      if (len - 5 > 0) {
+                         ND_PRINT((ndo, ", Identity: "));
+                         safeputs(ndo, data + 5, len - 5);
+                      }
+                   break;
+
+                 case EAP_TYPE_NOTIFICATION:
+                      if (len - 5 > 0) {
+                         ND_PRINT((ndo, ", Notification: "));
+                         safeputs(ndo, data + 5, len - 5);
+                      }
+                   break;
+
+                 case EAP_TYPE_NAK:
+                      count = 5;
+                     /*
+                      * one or more octets indicating the desired authentication
+                      * type one octet per type
+                      */
+                      while (count < len) {
+                         ND_PRINT((ndo, " %s (%u),",
+                                   tok2str(eap_type_values, "unknown", *(data+count)),
+                                   *(data + count)));
+                         count++;
+                      }
+                   break;
+
+                 case EAP_TYPE_TTLS:
+                      ND_PRINT((ndo, " TTLSv%u",
+                             EAP_TTLS_VERSION(*(data + 5)))); /* fall through */
```

Comment:
There is no `break`, if this fall-through is intentional, it should be decorated with a `ND_FALL_THROUGH` and possibly a comment (see commit eebb3f4 for an example).

Reply:
Have just seen the comment at the end of the line -- OK to leave as it is if you prefer.

Reply:
If I am the next reviewer of this change and I fail to understand **why** the code path falls through (besides just acknowledging the fact), I may comment on that again (or add the comment myself).